### PR TITLE
Admin member improvements

### PR DESCRIPTION
**Pull request by philipperolet**
Loading visual when members are loaded, cf screenshot (pulsating animation)--at dust we won't see it but Alan & Qonto might

and a few code improvements following daph suggestions and fix a modal not closing issue

Note: loading only shows when no members or no invitations (on screenshot I forced it to show)
image

### CODE REVIEW

**Comment by philipperolet** [comment]
As discussed IRL with @spolu, we switch to a "Cancel/save" type modal for user role change as for the rest of the UI, and wait for the list to update before closing the modal

File: front/pages/w/[wId]/members/index.tsx [hunk 1]
```diff
@@ -566,9 +604,21 @@ function ChangeMemberModal({
...
+      hasChanged={
+        selectedRole !== null && selectedRole !== member.workspaces[0].role
+      }
       title={member.name || "Unreachable"}
       type="right-side"
+      onSave={async () => {
+        setIsSaving(true);
+        // we know selectRole is not null because of the hasChanged check
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        await handleMemberRoleChange(member, selectedRole!);
```
Comment (spolu) [inline front/pages/w/[wId]/members/index.tsx:617]: We generally have our functions defined above the JSX. Can we move handleMemberRoleChange above?

Comment (spolu) [inline front/pages/w/[wId]/members/index.tsx:617]: Let's add a if clause instead of a eslint-disable ?

**Review by spolu — DISMISSED** [review]
Looks good, left a few minor commenrts

**Pull request merged**